aspnet/2-structured-data/Models/DatastoreBookStore.cs

```diff
@@ -29,6 +35,8 @@ public static class DatastoreBookStoreExtensionMethods
         /// <returns>A datastore key.</returns>
         public static Key ToKey(this long id)
         {
+            // P0 This function is especially useful, but I see no way to implement it

```

Comment:
Why not have a static key factory then? The keys produced here don't have a partition ID or kind - I'm surprised they don't cause the server to complain.

Note that key factories can be created without a client or anything - they need a root element, but that's all - so if you know the project ID, namespace ID and kind, you're done.